localstack-core/localstack/services/lambda_/event_source_mapping/pollers/sqs_poller.py

```diff
@@ -57,20 +63,34 @@ def get_queue_attributes(self) -> dict:
     def event_source(self) -> str:
         return "aws:sqs"
 
+    def receive_message(self, **kwargs) -> list[dict]:
```

Comment:
Wraps the `ReceiveMessage` operation with default arguments, making batch collection easier when passing this as an arg to the `message_collector` generator.